plugins/inputs/mesos/mesos.go

```diff
@@ -559,6 +561,24 @@ func getMetrics(role Role, group string) []string {
 			"registrar/state_store_ms/p999",
 			"registrar/state_store_ms/p9999",
 		}
+
+		m["overlay"] = []string{
```

Comment:
it might be nice to add a comment that links the reader to, e.g. https://github.com/dcos/dcos-mesos-modules/blob/master/overlay/master_metrics.cpp

same goes for the overlay/slave metrics